src/cmd/link/internal/arm64/obj.go

```diff
@@ -125,6 +126,10 @@ func archinit(ctxt *ld.Link) {
 		if *ld.FlagRound == -1 {
 			*ld.FlagRound = 0x10000
 		}
+
+	case objabi.Hwindows: /* PE executable */
```

Comment:
Should this have a TODO: or something so we can find it later?

Reply:
I don't think anything more is needed here. This case exists just to prevent the function from exiting due to "unknown" ctxt.HeadType.